packages/nameguard-sdk/src/index.test.ts

```diff
@@ -4,7 +4,7 @@ import { createClient, MAX_INSPECTED_NAME_CHARACTERS } from ".";
 
 const nameguard = createClient({
   // undefined will default to the production endpoint
-  endpoint: process.env.NAMEGUARD_API_URI,
+  nameguardEndpoint: process.env.NAMEGUARD_API_URI,
 });
 
 describe("NameGuard", () => {
```

Comment:
Suggest making a separate `describe(...` for each function we unit test and then make that function name the string passed to `describe(...`.